README.md

```diff
@@ -15,28 +15,30 @@
 Instantiate a new Angular module called `blog` that takes `ui.router` as a dependency.
 
 ```js
-// Your answer goes here...
+angular.module("blog", ["ui.router"])
 ```
 
 ### Question 2
 
 One button below has an `ng-click` attribute; the other has `data-ng-click` instead. What difference does it make?
 
+
+
 ```html
 <button ng-click="vm.create()">Click</button>
 <button data-ng-click="vm.create()">Click</button>
 ```
 
 ```text
-Your answer goes here...
+It does not make a difference, both will respond the same way.
```

Comment:
They'll respond the same way, but `data-` will make the code pass an HTML validator.